src/App/App.jsx

```diff
@@ -0,0 +1,19 @@
+import styles from './App.module.css'
+import success from '../assets/success.svg'
+import warning from '../assets/warning.svg'
+import error from '../assets/error.svg'
+import { Notifications } from '../components/Notifications';
+
+
+export const App = () => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.cardList}>
+        <Notifications variant="warning" message="Notification" icon={warning} className={styles.margin} color = "#FD9B00" />
+        <Notifications variant="success" message="Notification" icon={success} className={styles.margin} color = "#14B985"/>
+        <Notifications variant="error"   message="Notification"   icon={error}   className={styles.margin} color = "#FF3F54" />
```

Comment:
Треба зробити так, щоб потім полегшити собі життя! Навіщо icon і colo, якщо вже є variant) Ти ж можеш одразу задати і іконку і колір в залежності від типу variant 
також намагайся контент вставляти через children
```suggestion
        <Notifications variant="warning" className={styles.margin}>Notificcation</Notifications>
        <Notifications variant="success" className={styles.margin}>Notificcation</Notifications>
        <Notifications variant="error" className={styles.margin}>Notificcation</Notifications>
```